spec/integration/post_index_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+RSpec.describe 'Post index', type: :system do
+  user = User.first
+  post = Post.find(1)
+  comment = Comment.create(user:, post:, text: 'some text')
+  # it 'can see the users profile picture' do
+  #     visit user_post_path(user, post)
+  #     expect(page.body).to include('img_placeholder')
+  # end
+
+  it 'can see the current user name' do
+    visit user_post_path(user, post)
+    expect(page.body).to include(user.name)
+  end
+
+  it 'can see Number of Posts' do
+    visit user_posts_path(user)
+    expect(page.body).to include('Number of Posts: 4')
+  end
+
+  it 'can see post title/number' do
+    visit user_posts_path(user)
+    expect(page.body).to include("Post ##{post.id}")
+  end
+
+  it 'can see post text' do
+    visit user_posts_path(user)
+    expect(page.body).to include(post.text)
+  end
+
+  it 'can see the first comment' do
+    visit user_posts_path(user)
+    expect(page.body).to include(comment.text)
+  end
+
+  it 'can see the number of likes' do
+    visit user_posts_path(user)
+    expect(page.body).to include('Likes: 0')
+  end
+
+  it 'when I click on a post link, I am taken to the post show page' do
+    visit user_posts_path(user)
+    click_link("Post ##{post.id}")
+    expect(current_path).to eq user_post_path(user, post)
+  end
+end
```

Comment:
- [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md) for this project, " 
Login page:

- I can see the username and password inputs and the "Submit" button.
- When I click the submit button without filling in the username and the password, I get a detailed error.
- When I click the submit button after filling in the username and the password with incorrect data, I get a detailed error.
- When I click the submit button after filling in the username and the password with correct data, I am redirected to the root page ", please kindly create a new file and add more tests for the Login page.

Reply:
@t-yanick 

our project did not have a login page required. here is the wireframe https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md